vizro-core/docs/pages/development/contributing.md

```diff
@@ -1,6 +1,6 @@
 ## Contributing guidelines
 
-Contributions of all experience levels are welcome! There are many ways to contribute, and we appreciate all of them. Please use our [issues page](https://github.com/mckinsey/vizro/issues) to discuss any contributions.
+Contributions of all experience levels are welcome! There are many ways to contribute, and we appreciate all of them. Please use our [issues page](https://github.com/mckinsey/vizro/issues) to discuss any contributions. Please ensure to first open an issue to discuss a contribution before opening a pull request.
```

Comment:
Shall we detail out the steps? Similar to this contribution guide for example: https://github.com/plotly/plotly.js/blob/master/CONTRIBUTING.md

1. Issue/Discussion
2. Iteration
3. Approval
4. Development
5. Review

If it's an overkill, then ignore the comment. I just like their contribution guide as it details out what they can expect as well in terms of responses etc.

Reply:
Puh, I think it is a bit overkill - maybe in the future we can expand, but let's get some contributions first :)